tests/phpunit/SnowflakeTest.php

```diff
@@ -81,18 +81,28 @@ public function testDefaultWarehouse(): void
 
             // run with warehouse param
             $config = $this->getConfig();
-            $app = new SnowflakeApplication($config, new Logger(), [], $this->dataDir);
+            JsonHelper::writeFile($this->dataDir . '/config.json', $config);
 
-            $result = $app->run();
-            $this->assertEquals('success', $result['status']);
-            $this->assertCount(3, $result['imported']);
+            $logger = new TestLogger();
+            $app = new SnowflakeApplication($logger);
+
+            $app->execute();
+
+            Assert::assertTrue(
+                $logger->hasInfo('Downloading data from Snowflake.')
+            );
+            Assert::assertTrue(
+                $logger->hasWarning('Query result set is empty. Exported "0" rows to "in.c-main.tableColumns".')
```

Comment:
Nebyl předtím nějaký result? Nevím, co znamená `$this->assertCount(3, $result['imported']);` 🤷‍♂️

Reply:
`imported` byly 3 stažené tabulky :D

Reply:
nebylo to nic o tom kolik se stáhlo záznamů